flink/v1.14/flink/src/main/java/com/netease/arctic/flink/read/FlinkSplitPlanner.java

```diff
@@ -75,6 +94,27 @@ private static List<ArcticSplit> planFullTable(
     return allSplits;
   }
 
+  public static List<ArcticSplit> mergeOnReadPlan(
```

Comment:
I suggest adding some Javadoc here to tell the function of the method, especially the difference with `planFullTable`.